kite-morphlines/kite-morphlines-core/pom.xml

```diff
@@ -62,6 +62,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+    	<groupId>commons-codec</groupId>
+    	<artifactId>commons-codec</artifactId>
```

Comment:
Resolved by shading the relevant methods into a static class. Shading the whole classes had too many dependencies.